### DISCUSSION TITLE
Pulling titles into content doesn't work

### DISCUSSION THREAD

**Discussion by AngelinaPukhova** [requarks/wiki#7228] [Error / Bug Report]
Pulling headings into content does not work. In the text you make headings 1, 2, 3, etc. As a result, only the 1st and 2nd level of headings are displayed in the content, the rest are ignored and not displayed. please fix it urgently.
![header 123](https://github.com/requarks/wiki/assets/173324221/8fcc0e32-a1bb-4fd6-ae65-05779571fdf6)

**Comment by fnafox** [comment]
This is intended behavior.

**Reply by AngelinaPukhova** [reply]
So it should display all levels of headings in the table of contents, and not just headings 1 and 2, otherwise why do the other 4 headings even exist? Please fix this bug, it interferes with work. Employees cannot find the required section (subsection) in a short time, which slows down the work process.

**Reply by AngelinaPukhova** [reply]
please solve this problem as soon as possible.
